02 - Community Expansions/02.01 Plugins by Category/Plugins to manage files and attachments.md

```diff
@@ -7,28 +7,21 @@ publish: true
 ---
 
 
-# File
+# Plugins to manage files and attachments
```

Comment:
"manage files" is a rather broad category where quite a lot more plugins would fit. I think "attachments" alone might suffice?